### DISCUSSION TITLE
error in my code

### DISCUSSION THREAD

**Discussion by navidanchitrali** [tekartik/sqflite#666] [Q&A]
Hello sir!
When i run the funtions i get an error saying, 'Undefined name factory', please help!

// First version of the database
  db = await factory.openDatabase(path,
  options: OpenDatabaseOptions(
  version: 1,
  onCreate: (db, version) async {
  var batch = db.batch();
  _createTableCompanyV1(batch);
  await batch.commit();

**Comment by alextekartik** [comment]
There is no such definition. Try:
```
var factory = databaseFactory;
```
